index.html

```diff
@@ -0,0 +1,199 @@
+                  <li class="separator border"></li>
+                  <li class="nav-item"><a class="nav-link text-dark font-weight-bold" href="#">Tech &amp; Science</a></li>
+                  <li class="separator border"></li>
+                  <li class="nav-item"><a class="nav-link text-dark font-weight-bold" href="#">Culture</a></li>
+                  <li class="separator border"></li>
+                  <li class="nav-item"><a class="nav-link text-dark font-weight-bold" href="#">Newsgeek</a></li>
+                  <li class="separator border"></li>
+                  <li class="nav-item"><a class="nav-link text-dark font-weight-bold" href="#">Sports</a></li>
+                  <li class="separator border"></li>
+                  <li class="nav-item"><a class="nav-link text-dark font-weight-bold" href="#">Health</a></li>
+                  <li class="separator border"></li>
+                  <li class="nav-item"><a class="nav-link text-dark font-weight-bold" href="#">The Debate</a></li>
+                  <li class="separator border"></li>
+                  <li class="nav-item"><a class="nav-link text-dark font-weight-bold" href="#">Experts</a></li>
+                  <li class="separator border"></li>
+                  <li class="nav-item"><a class="nav-link text-dark font-weight-bold" href="#">Vantage</a></li>
+                  <li class="icon-input nav-item ">
+                    <input class="border-0 w-100 nav-link text-dark font-weight-bold" type="text" placeholder="Search">
+                    <span><i class="fas fa-search"></i></span>
+                  </li>
+                </ul>
+                <div class="trapezoid d-none d-lg-block"></div>
+              </div>
+            </nav>
+            <div class="white-space bg-white border-bottom d-none d-lg-block"></div>
+      
+            <ul class="secondary-links nav bg-white font-weight-bolder d-none d-lg-flex px-3 align-items-center flex-nowrap border-bottom border-top">
+              <li class="nav-item"><a class="nav-link text-dark font-weight-bold" href="#">TRENDING</a></li>
+              <li class="nav-item"><a class="nav-link text-secondary pr-5" href="#">JOE BIDEN</a></li>
+              <li class="nav-item"><a class="nav-link text-secondary pr-5" href="#">DONALD TRUMP</a></li>
+              <li class="nav-item"><a class="nav-link text-secondary pr-5" href="#">CORONAVIRUS</a></li>
+              <li class="nav-item"><a class="nav-link text-secondary pr-5" href="#">STIMULUS PACKAGE</a></li>
+              <li class="nav-item"><a class="nav-link text-secondary pr-5" href="#">VACCINE</a></li>
+              <li class="nav-item"><a class="nav-link text-secondary pr-5" href="#">CAPITOL RIOTS</a></li>
+            </ul>
+          
+
+        </div>
+
+      </header>
+
+    
+    <main>
+        <div class="container-fluid">
+            <div class=" main-content w-100  h-50 bg-light" ><h1>Main content</h1></div>
+            </div>
+    </main>
+    <footer>
```

Comment:
- [ ] Your footer should stick to the bottom of the page even if there is no content. You can make your footer stick to the bottom by adding the following bootstrap class `fixed-bottom`
```suggestion
    <footer class='fixed-bottom'>
```

Reply:
updated as requested